src/sentry/quotas/base.py

```diff
@@ -491,3 +496,34 @@ def get_transaction_sampling_tier_for_volume(
         :param organization_id: The organization id.
         :param volume: The volume of transaction of the given project.
         """
+
+    def assign_seat(
+        self,
+        monitor: Monitor,
+    ) -> int:
+        """
+        Determines if a monitor seat assignment is accepted or rate limited. The Monitor status
+        will be updated from ACTIVE to OK if the seat assignment is accepted.
+        """
+        from sentry.monitors.models import MonitorStatus
+        from sentry.utils.outcomes import Outcome
+
+        monitor.update(status=MonitorStatus.OK)
+        return Outcome.ACCEPTED
+
+    def unassign_seat(
+        self,
+        monitor: Monitor,
+    ):
+        """
+        Disables a monitor seat assignment and sets the Monitor status to DISABLED
+        """
+        from sentry.monitors.models import MonitorStatus
+
+        monitor.update(status=MonitorStatus.DISABLED)
+
+    def set_billing_seat_recreate(self, monitor: Monitor):
+        """Sets the monitor's seat assignment to automatically be recreated at renewal."""
+
+    def remove_billing_seat_recreate(self, monitor: Monitor):
+        """Removes the monitor's seat assignment so it is NOT automatically be recreated at renewal."""
```

Comment:
Yeah I think this definitely should be part of some type of getsentry service no?

Reply:
My thought was to utilize Quotas since it impacts quota usage. Is there a getsentry service that would be better?